stylesheets/styles.css

```diff
@@ -0,0 +1,209 @@
+/* --- GLOBAL --- */
+
+* {
+  box-sizing: border-box;
+}
+
+/* --- PRIMARY LAYOUT --- */
+
+/* header, navigation bar, navigation links */
+
+header {
+  height: 6em;
+  padding-top: 0.5em;
+  padding-right: 0.5em;
+  margin-top: 2%;
+  background-color: rgba(200, 194, 208, 0.4);
+  border-radius: 1.5em;
+}
+
+nav ul {
+  list-style-type: none;
+  margin: 2%;
+  padding: 0;
+  font-family: Verdana, Arial;
+  font-size: 1.5em;
+}
+
+
+nav li {
+  display: inline;
```

Comment:
The display property actually doesn't work on floated elements. Having this here doesn't break anything, but it also doesn't do anything, so I'd pull it out.